skills.cpp

```diff
@@ -221,12 +221,6 @@ int StudyRateAdjustment(int days, int exp)
 	return rate;
 }
 
-ShowSkill::ShowSkill(int s, int l)
```

Comment:
I made this into a struct with static initialization because it was a lot clearer.